app/src/main/java/ch/epfl/sdp/blindly/audio/RecordingActivity.kt

```diff
@@ -60,7 +60,7 @@ class RecordingActivity : AppCompatActivity(), AudioLibraryAdapter.OnItemClickLi
 
     private var totalNumberOfRec = 0
 
-    private var maximumAudioDuration = DEFAULT_RECORD_AUDIO_DURATION
+    private var maximumAudioDuration = DEFAULT_RECORD_AUDIO_DURATION;
```

Comment:
Is the `;` needed, on my side studio tends to complain about them

Reply:
Nope not at all, I must have merged the wrong of the two lines...